Italian/guardare/vecchie-fotografie/girare_foto/messaggio.md

```diff
@@ -0,0 +1,5 @@
+Giri le fotografie e dietro ad una trovi uno strano messaggio in parte cancellato dal tempo.
+
+La fotografia ritrae una famiglia numerosa, tutti i volti sono ben visibili ad eccezione di quello di una giovane ragazza che è stato bruciato.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.